supertokens_python/recipe/session/framework/django/syncio/__init__.py

```diff
@@ -67,10 +67,10 @@ def wrapped_function(request: HttpRequest, *args: Any, **kwargs: Any):
                     baseRequest.set_session(session)
                 return f(baseRequest.request, *args, **kwargs)
             except SuperTokensError as e:
-                response = DjangoResponse(JsonResponse({}))
+                response_ = DjangoResponse(JsonResponse({}))
```

Comment:
Why was this renamed?

Reply:
Because of response var that I introduced. Not required anymore.

Removed.